content/blog/2017/2017-02-10-jenkins-datadog-plugin.adoc

```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title: "Monitor Jenkins jobs with the Datadog plugin"
+tags:
+- plugins
+- monitoring
+author: echang26
+---
+
+[NOTE]
+====
+A modified version of this article was originally posted on the
```

Comment:
Please also include in this admonition block something like "This is a guest post by Emily Chang, so and so at Datadog. A modified version of this article was originally published over here"